### DISCUSSION TITLE
Micrsoft Windows Support???

### DISCUSSION THREAD

**Discussion by flyingosprey** [neutrinolabs/xrdp#2802] [Q&A] [answered]
Currently, it only runs on Linux/Mac. Any chance for "Micrsoft Windows Support"?

I have this feature request because RDP service is only avaialbe on Windows Server/Enterprise/Pro SKU.

But majority of Windows PC are "Windows Home" SKU, where no RDP service exists on them.

Thanks.

**Comment by metalefty** [comment]
I'm afraid I have no plan to support Windows yet. Windows have quite different system architecture from Linux, massive volume of development will be required and we don't have such resource.

**Comment by matt335672** [comment, marked as answer]
We've had this discussion @flyingosprey, although it's difficult to find. See #2100, in particular my [comment](https://github.com/neutrinolabs/xrdp/issues/2100#issuecomment-1004655307) on it which contains a link you might be interested in.
